src/application/common/logger.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2017 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as Logger from 'bunyan';
+
+const ILogger = Symbol("ILogger");
+
+interface ILogger {
```

Comment:
nope, it is very subtle, anything can be a valid JSON object if you know what you are doing, like if you are fine that functions are omitted